pkgs/top-level/all-packages.nix

```diff
@@ -2982,9 +2982,7 @@ with pkgs;
 
   dpic = callPackage ../tools/graphics/dpic { };
 
-  dragon-drop = callPackage ../tools/X11/dragon-drop {
-    gtk = gtk3;
-  };
+  dragon-drop = throw "'dragon-drop' has been renamed to/replaced by 'xdragon'"; # Converted to throw 2022-03-27;
```

Comment:
Maybe put a `dragon-drop = xdragon; # Added 2022-03-27` line in `pkgs/top-level/aliases.nix` instead? We have [scripts](https://github.com/NixOS/nixpkgs/blob/master/maintainers/scripts/remove-old-aliases.py) to convert alias to throw.

Reply:
Fixed

Reply:
On one hand, I agree with @bobby285271. On the other hand, the package about the be dropped provides a binary called `dragon`, whereas the one that will remain provides a binary called `xdragon`, for reasons outlined in https://github.com/NixOS/nixpkgs/pull/95925 or rather https://github.com/mwh/dragon/issues/17

So throw might be the better option, after all.

Reply:
I also think throwing maybe a better option in this case.